src/components/Header/index.js

```diff
@@ -13,8 +13,8 @@ import NavigationTabs from "../NavigationTabs";
 
```

Comment:
Can you update CipherLogo as well on line 7?

Here is the link to the logo: https://files.readme.io/b56e3b4-small-icon-512px.png

Please download it as png, and put in `assets/images/`

Reply:
done!